mismatched signatures between the .m and .h files

--- origsrc/rigs-0.2.2/Source/RIGSProxySetup.h	2001-09-24 06:24:46.000000000 -0500
+++ src/rigs-0.2.2/Source/RIGSProxySetup.h	2009-08-03 02:12:02.521829500 -0500
@@ -90,7 +90,7 @@ struct _RIGSSelectorIDTable
  * objective-C runtime. 
  */
 int _RIGS_ruby_method_arity(const char *rb_class_name, const char *rb_mth_name);
-id  _RIGS_register_ruby_class (VALUE rb_class);
+Class _RIGS_register_ruby_class (VALUE rb_class);
 VALUE _RIGS_register_ruby_class_from_ruby (VALUE self, VALUE rb_class);
 BOOL _RIGS_build_objc_types(VALUE rb_class, const char *rb_mth_name,
 			    const char retValueType, int nbArgs, char *sigBuf);
